src/fmu/settings/models/project_config.py

```diff
@@ -12,11 +12,49 @@
 from fmu.settings.types import ResettableBaseModel, VersionStr  # noqa: TC001
 
 
+class RmsCoordinateSystem(BaseModel):
+    """The project coordinate system of an RMS project."""
+
+    name: str
+    """Name of the coordinate system."""
+
+
+class RmsStratigraphicZone(BaseModel):
+    """A stratigraphic zone from an RMS project."""
+
+    name: str
+    """Name of the zone."""
+
+    top: str
+    """Name of the horizon at the top of the zone."""
+
+    base: str
+    """Name of the horizon at the base of the zone."""
```

Comment:
I think we should add some more context through the naming of these fields as top/base doesn't convey much. Either `top_horizon` or `top_horizon_name` would improve it

```suggestion
    top_horizon_name: str
    """Name of the horizon at the top of the zone."""

    base_horizon_name: str
    """Name of the horizon at the base of the zone."""
```

Reply:
okay i like `top_horizon_name`, will change it to that